src/js/components/stock-movement-wizard/AddItemsPage.jsx

```diff
@@ -401,6 +425,10 @@ class AddItemsPage extends Component {
         oldItem.recipient.id : oldItem.recipient;
       const newRecipient = item.recipient && _.isObject(item.recipient) ?
         item.recipient.id : item.recipient;
+      const duplicatedItem = _.find(
+        lineItemsToBeAdded,
```

Comment:
Why you look into `lineItemsToBeAdded`

Reply:
We are searching for duplicated item in newly created table with items to add because there might be more than two items with the same product in newItems table.